lib/tpm2_eventlog.h

```diff
@@ -17,23 +17,25 @@ typedef bool (*EVENT2DATA_CALLBACK)(TCG_EVENT2 const *event, UINT32 type,
                                     void *data);
 typedef bool (*SPECID_CALLBACK)(TCG_EVENT const *event, void *data);
 
+typedef struct {
+    void * data;
+    SPECID_CALLBACK specid_cb;
+    EVENT2_CALLBACK event2hdr_cb;
+    DIGEST2_CALLBACK digest2_cb;
+    EVENT2DATA_CALLBACK event2_cb;
+    uint8_t sha1_pcrs[TPM2_MAX_PCRS][TPM2_SHA1_DIGEST_SIZE];
+    uint8_t sha256_pcrs[TPM2_MAX_PCRS][TPM2_SHA256_DIGEST_SIZE];
+} tpm2_eventlog_ctx_t;
```

Comment:
We also don't do this _t stuff. If it's in the type location for the language it's obviously a type.

Reply:
Renamed to `tpm2_eventlog_context`.